proposals/0186-loaded-transaction-data-size-specification.md

```diff
@@ -114,6 +114,35 @@ Account size for programs owned by LoaderV4 is left undefined. This SIMD should
 be amended to define the required semantics before LoaderV4 is enabled on any
 network.
 
+### Sidebar: Program ID and Loader Validation
+
+After all accounts are loaded, if the transaction's size comes at or under its
+loaded accounts data size limit, the program IDs of each instruction, along with
+their program owners, are validated. This process is unrelated to the rest of
+this SIMD, but is not defined elsewhere, so we define it here.
```

Comment:
I think it's fair to say that it *is* related to this SIMD because now that we don't need to load the owner account to track its size, we are able to make this simplification.

Reply:
ill reword it, im trying to adhere to the philosophy that the simd document is primary and the solana validators are concrete implementations of the simd. its unrelated in the sense that this simd is exclusively about how sizes are determined so the validation is sort of a tangent